Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityDeviceManager.cs

```diff
@@ -115,12 +111,49 @@ private WindowsMixedRealityController GetOrAddController(InteractionSourceState
                     throw new ArgumentOutOfRangeException();
             }
 
-            var inputSource = InputSystem?.RequestNewGenericInputSource($"Mixed Reality Controller {controllingHand}");
+            var pointers = new List<IMixedRealityPointer>();
```

Comment:
Question, could this be part of the inputSource request?
rather than put this in every device we create, effectively copying the same code.
Or failing that, move this to a static function to return configured pointers for a given controller / type and hand

Reply:
Possibly I'll see what I can do.

Reply:
Doesn't look like we can embed this into the input source request. There's too many controller specific things that could end up happening while we register our pointers. \

First being that we need to know the type of controller.

Second, in WMR we can query if the input source pose supports pointing or not.

We would also need to know the handedness of the controller and check against the list of pointers in our pointer profile that match the handedness of our controllers.

We could make all those parameters that the input source request makes, but then I think that breaks the OOP nature of the setup, and makes the request do more than it's responsible for.

Reply:
I figured something out, I think you'll like the result.